TheraLang.BLL/Infrastructure/BllServiceCollectionExtensions.cs

```diff
@@ -24,11 +25,18 @@ public static IServiceCollection AddUnitOfWork(this IServiceCollection services)
             return services.AddTransient<IUnitOfWork, UnitOfWork>(provider =>
                 new UnitOfWork(provider.GetRequiredService<IttmmDbContext>()));
         }
-        public static IServiceCollection AddAzureStorageClientFactory(this IServiceCollection services, string connectionString)
+        
+        public static IServiceCollection AddFileStorage(this IServiceCollection services, string connectionString)
         {
-            return services.AddTransient<IAzureConnectionFactory, AzureConnectionFactory.AzureConnectionFactory>(serviceProvider =>
+            if (string.IsNullOrWhiteSpace(connectionString))

```

Comment:
how about making a choice of file storage dependant on HostingEnvironment?

Reply:
It still need to be tested locally in dev mode. Checking conn string will allow to dynamiccally change storage